openedx/features/teams/tests/test_serializers.py

```diff
@@ -68,10 +68,15 @@ def setUp(self):
 class CreateTeamsSerializerTestCase(SerializersTestBaseClass):
     """ Tests for the create team serializer."""
 
+    def _get_team_dict_data(self):
+        data = self.team.__dict__
```

Comment:
see if this works `dict(self.team)`

Reply:
It did not work  `TypeError: 'CourseTeam' object is not iterable`